app/code/Magento/PaypalGraphQl/etc/schema.graphqls

```diff
@@ -141,3 +142,7 @@ input PayflowProResponseInput @doc(description:"Input required to complete payme
 type PayflowProResponseOutput {
     cart: Cart!
 }
+
+type StoreConfig {
+    payflowpro_vault_enabled: String @doc(description: "Payflowpro vault status.")
```

Comment:
@keharper @joni-jones I added a store config field for vault. Let me know if the name/description needs to be updated.

Reply:
@prabhuram93, my understanding that for `StoreConfig` values, we follow the same semantic as in `config.xml`. For Payflow Pro it will be `payment_payflowpro_cc_vault_active`.

cc @paliarush

Reply:
@joni-jones We were doing that until we implemented aliasing. We can have custom field names that are not tied to config.xml. But if you feel ```payment_payflowpro_cc_vault_active``` is more relevant I can update accordingly.

Reply:
Since we have aliases now, let's use whatever makes the most sense from the API client perspective. We also need to make sure to have proper namespaces to prevent conflicts in the future.

Reply:
Thanks @paliarush ```payment_payflowpro_cc_vault_active``` has better namespacing. @joni-jones I am gonna go with that.